app/javascript/packs/application.js

```diff
@@ -1,13 +1,14 @@
-import 'admin-lte/bootstrap/css/bootstrap.css';
```

Comment:
Note: Importing bootstrap.css here is unnecessary, and also it sometimes leads to double loading errors. So I just removed this line.

- AdminLTE itself loads its bundled bootstrap.css automatically, as https://adminlte.io/docs/3.2/upgrade-guide.html says "You will not need to load the Bootstrap CSS file bootstrap(.min).css, but you will need the Bootstrap JS file bootstrap(.min).js."
- One of AdminLTE dependencies, bootstrap-switch, depends on bootstrap v3 (not v4). Hence node_modules/bootstrap is v3, and node_modules/admin-lte/node_modules/bootstrap is v4.
- I haven't checked out why, but importing node_modules/admin-lte/node_modules/bootstrap/dist/css/bootstrap.css manually causes conflicts of style.